app/views/announcements/blocks/_donation_goal.html.erb

```diff
@@ -12,7 +12,9 @@
       </p>
       <div class="bg-gray-200 dark:bg-neutral-700 rounded-full w-full">
         <div class="h-full bg-primary rounded flex items-center justify-center" style="width: <%= percentage * 100 %>%">
-          <p class="text-sm text-black p-[1px] my-0"><%= number_with_precision(percentage * 100, precision: 1) %>%</p>
+          <p class="text-sm text-black p-[1px] my-0">
+            <%= percentage > 0 ? "#{number_with_precision(percentage * 100, precision: 1)}%" : "&nbsp".html_safe %>
+          </p>
```

Comment:
What are you trying to achieve with the `&nbsp`?

Reply:
It was to add whitespace to keep the height of the progress bar, but I changed it to use just a fixed height instead